### PR TITLE
Feature/cplp 2061/app details privacy policy

### PR DESCRIPTION
**Pull request by lavanya-bmw**

### CODE REVIEW

File: cx-portal/src/components/shared/basic/ReleaseProcess/AppMarketCard/index.tsx [hunk 1]
```diff
@@ -333,12 +335,12 @@ export default function AppMarketCard() {
...
-      leadPictureUri:
-        data.uploadImage.leadPictureUri !== null &&
-        Object.keys(data.uploadImage.leadPictureUri).length > 0 &&
-        Object.values(data.uploadImage.leadPictureUri)[0],
       salesManagerId: salesManagerId,
-      useCaseIds: data.useCaseCategory,
+      useCaseIds: data.useCaseCategory.some((value) => {
+        return typeof value == 'object'
+      })
+        ? data.useCaseCategory?.map((item: any) => item.id)
```
Comment (manojava-gk) [inline cx-portal/src/components/shared/basic/ReleaseProcess/AppMarketCard/index.tsx:342]: Do not use `any`

**Review by manojava-gk — COMMENTED** [review]
Data type issue

File: cx-portal/src/components/shared/basic/ReleaseProcess/AppMarketCard/index.tsx [hunk 2]
```diff
@@ -333,12 +335,12 @@ export default function AppMarketCard() {
...
       provider: data.provider,
-      leadPictureUri:
-        data.uploadImage.leadPictureUri !== null &&
-        Object.keys(data.uploadImage.leadPictureUri).length > 0 &&
-        Object.values(data.uploadImage.leadPictureUri)[0],
       salesManagerId: salesManagerId,
-      useCaseIds: data.useCaseCategory,
+      useCaseIds: data.useCaseCategory.some((value) => {
+        return typeof value == 'object'
+      })
```
Comment (oyo) [inline cx-portal/src/components/shared/basic/ReleaseProcess/AppMarketCard/index.tsx:341]: when the only statement of an arrow function is return we can simplify this easier to read as:
`data.useCaseCategory.some((value) => typeof value === object`

File: cx-portal/src/components/shared/basic/ReleaseProcess/AppMarketCard/index.tsx [hunk 3]
```diff
@@ -359,7 +361,10 @@ export default function AppMarketCard() {
       ],
       supportedLanguageCodes: data.appLanguage,
       price: data.price,
-      privacyPolicies: [],
+      privacyPolicies: data.privacyPolicies,
+      providerUri: appStatusData?.providerUri || '',
+      contactEmail: appStatusData?.contactEmail || '',
+      contactNumber: appStatusData?.contactNumber || '',
```
Comment (oyo) [inline cx-portal/src/components/shared/basic/ReleaseProcess/AppMarketCard/index.tsx:367]: for default values the use of `??` is preferred over `||`

**Review by oyo — CHANGES_REQUESTED** [review]
some remarks

File: cx-portal/src/components/shared/basic/ReleaseProcess/AppMarketCard/index.tsx [hunk 4]
```diff
@@ -82,6 +82,7 @@ type FormDataType = {
     alt?: string
   }
   salesManagerId: string | null
+  privacyPolicies: string[] | []
```
[the remark this answers is not in the document]
Comment (lavanya-bmw) [reply cx-portal/src/components/shared/basic/ReleaseProcess/AppMarketCard/index.tsx]: Done

File: cx-portal/src/components/shared/basic/ReleaseProcess/AppMarketCard/index.tsx (hunk 3 above)
Comment (lavanya-bmw) [reply cx-portal/src/components/shared/basic/ReleaseProcess/AppMarketCard/index.tsx]: Done

**Review by oyo — APPROVED** [review]

**Pull request merged**